### ISSUE TITLE
Something missing in course 2

### ISSUE THREAD

**Issue by deladan** [remisharrock/c-programming-with-linux-MOOC-issues-tracker#660]
I think it's already been reported, but in unit 2.1,  but it would be great if some example of use of embedded if/else statements were added to codecast lectures.
Several learners were lost when having to deal with several if/else instructions in exercises such as the one about hostel cost.

**Comment by EllaHa** [comment]
Thanks Isabelle for this report. @AadhyaKocha Can you please Modify the statement to add the following:
Please use the logical and and the logical or introduced in the previous section. 

For this activity
https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.02+1T2020/courseware/a86febae1149479d971ae6edbd850c51/fc9008c149444ca0a623cc332fc01e06/?child=first

**Comment by deladan** [comment]
Hello @EllaHa ,
Issue is that even solution doesn't use these logical operators but several if/else statements.

**Comment by remisharrock** [comment]
@deladan that is right for https://github.com/remisharrock/cspace/blob/master/02%20C%20Programming%20Language%20Foundations/Unit%202.1%20Logic%20conditions%20and%20if-else%20statements/03%20Combining%20logic%20conditions%20using%20the%20logical%20AND%20and%20OR/03_thostelAge.c    but we were working on  https://github.com/remisharrock/cspace/blob/master/02%20C%20Programming%20Language%20Foundations/Unit%202.1%20Logic%20conditions%20and%20if-else%20statements/01%20Using%20the%20if%20statement/06_thostel.c   
Indeed we should add an explanation: that is why issue #663 is now related 👍

**Comment by deladan** [comment]
Hello Rémi,

I'm sorry, but I don't understand.
The activity about [costly hotel rooms](https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.02+1T2020/courseware/a86febae1149479d971ae6edbd850c51/fc9008c149444ca0a623cc332fc01e06/?child=last), which is in the section about or and and logical operators, doesn't require those operators.
Indeed, there should be an example about embedded if/else statements, but I don't understand why we would ask the learner to use or and and operators in her code if even provided solution doesn't use them.
For this activity, I suggest we add a warning saying that this is another problem about if/else statement, not about logical operators.
A lot of learners reported that they were confused by this, as they were trying to add logical operators uselessly.
For instance: [https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.02+1T2020/discussion/forum/11dde5d350ab3ff7a9e35ce851a7a5de72b7fd12/threads/5fe03deacfe4fd080b24f945](https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.02+1T2020/discussion/forum/11dde5d350ab3ff7a9e35ce851a7a5de72b7fd12/threads/5fe03deacfe4fd080b24f945)

**Comment by deladan** [comment]
Another [post](https://courses.edx.org/courses/course-v1:Dartmouth_IMTx+DART.IMT.C.02+1T2020/discussion/forum/11dde5d350ab3ff7a9e35ce851a7a5de72b7fd12/threads/5ffa94809c2aee097ddda1c6) about this confusion.
I really think some note is added to problem statement to warn the learner the activity isn't about the use of logical operators.
Even activity's name is confusing: complex logic conditions to find hostel cost.
Ideally, this problem should be replaced with one which requires those logical operators.

**Comment by deladan** [comment]
Some learner suggests some hint were added to the exercise, saying that nested if/else statements are possible.
Maybe this could be added without a lot of work, until some example is given in the course's lectures.
